src/components/TodoList/TodoList.tsx

```diff
@@ -1,13 +1,33 @@
 /* eslint-disable max-len */
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from '../../app/hooks';
+import { actions as currentTodoActions } from '../../features/currentTodo';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => {
-  return (
-    <>
+type PropTypes = {
+  todos: Todo[]
+};
+
+export const TodoList: React.FC<PropTypes> = ({ todos }) => {
+  const dispatch = useDispatch();
+
+  const currentTodo = useAppSelector(state => state.currentTodo);
```

Comment:
try to avoid using useAppSeletor in every component, you can just take currentTodo as a prop from App.tsx